### ISSUE TITLE
AWX External logging in AWX 10.0.0 doesn't work

### ISSUE THREAD

**Issue by anksvault** [ansible/awx#6726]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Post migrating AWX 9.2.0 to AWX 10.0.0, Splunk logging which was configured has stopped working. When trying to run test logging in AWX 10.0.0, it gives following error:

![image](https://user-images.githubusercontent.com/42296733/79436492-95c6d000-7fee-11ea-9ac8-6c9a04816ee3.png)

AWX Web Container logs give the following messages when the test is executed.

```
2020-04-16 08:44:26,864 WARNING  awx.api.generics status 401 received by user AnonymousUser attempting to access /api/v2/settings/logging/test/ from 10.204.230.37
`background_callback` is deprecated and will be removed in 1.0, use `hooks` instead
2020-04-16 08:44:26,875 WARNING  django.request Bad Request: /api/v2/settings/logging/test/
[pid: 371|app: 0|req: 880/18930] 10.204.230.37 () {64 vars in 2972 bytes} [Thu Apr 16 08:44:26 2020] POST /api/v2/settings/logging/test/ => generated 49 bytes in 166 msecs (HTTP/1.1 400) 8 headers in 234 bytes (1 switches on core 0)
```

##### ENVIRONMENT
* AWX version: 10.0.0
* AWX install method: docker on linux
* Ansible version:  2.9.6
* Operating System: RHEL 7
* Web Browser: Google Chrome 81.0.4044.113 (Official Build) (32-bit)

##### STEPS TO REPRODUCE

From AWX 10.0.0, click: SETTINGS > SYSTEM > LOGGING > Fill Splunk/Logging Aggregator details > Save and Click Test

##### EXPECTED RESULTS

The test should be successful and test message should be sent to External Logging tool.

##### ACTUAL RESULTS
**UI Results**
```
**ERROR!
There was an error testing the log aggregator. Returned status: 400 JSON parse error - not a JSON object**
```

**Container Results**
```
2020-04-16 08:44:26,864 WARNING  awx.api.generics status 401 received by user AnonymousUser attempting to access /api/v2/settings/logging/test/ from 10.204.230.37
`background_callback` is deprecated and will be removed in 1.0, use `hooks` instead
2020-04-16 08:44:26,875 WARNING  django.request Bad Request: /api/v2/settings/logging/test/
[pid: 371|app: 0|req: 880/18930] 10.204.230.37 () {64 vars in 2972 bytes} [Thu Apr 16 08:44:26 2020] POST /api/v2/settings/logging/test/ => generated 49 bytes in 166 msecs (HTTP/1.1 400) 8 headers in 234 bytes (1 switches on core 0)
```

##### ADDITIONAL INFORMATION

![image](https://user-images.githubusercontent.com/42296733/79436461-89db0e00-7fee-11ea-8aa4-e06b7ba16c8f.png)

**Comment by ryanpetrello** [comment]
@anksvault,

Any chance you can share the HTTP POST body content for the request your browser is making?

**Comment by anksvault** [comment]
@ryanpetrello 
Not sure this is what is required.

![image](https://user-images.githubusercontent.com/42296733/79472485-87de7280-8021-11ea-9ac9-967c45db8e34.png)

**Comment by ryanpetrello** [comment]
Hey @anksvault,

Mind giving this a shot on the (just released) 11.0.0?  We actually *just* released this this morning, and it includes an entirely new implementation of AWX's external logging support:

https://github.com/ansible/awx/issues/5155

**Comment by ifeelcream** [comment]
I had the same error and upgrading to 11.0.0 fixed it for me.
